### PR TITLE
rsavu98/KeepHydrated

### PR DESCRIPTION
**Pull request by lhalam**

### CODE REVIEW

**Review by rsavu98 — DISMISSED** [review]

File: src/com/ss/ita/kata/implementation/rsavu98/SixImpl.java [hunk 1]
```diff
@@ -3,4 +3,49 @@
...
+    @Override
+    public double twoDecimalPlaces(double number) {
+        return 0;
+    }
+
+    @Override
+    public int[] divisibleBy(int[] numbers, int divider) {
+        return new int[0];
+    }
+}
```
Comment (lhalam) [inline src/com/ss/ita/kata/implementation/rsavu98/SixImpl.java]: add endline

**Review by lhalam — COMMENTED** [review]
resolve conflict

File: src/com/ss/ita/kata/implementation/rsavu98/SixImpl.java (hunk 1 above)
Comment (rsavu98) [reply src/com/ss/ita/kata/implementation/rsavu98/SixImpl.java]: @lhalam Thank you! Endline is added.

**Review by Udzhas — DISMISSED** [review]

**Review by BohdanDziubailo — DISMISSED** [review]

**Comment by rsavu98** [comment]
This pull request is no longer needed. These changes were merged in another commit.

**Pull request closed without merging**